edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudConnectionProvider.cs

```diff
@@ -16,8 +16,6 @@ namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy
 
     public class CloudConnectionProvider : ICloudConnectionProvider
     {
-        // Minimum value allowed by the SDK for Connection Idle timeout for AMQP Multiplexed connections.
-        static readonly TimeSpan MinAmqpConnectionMuxIdleTimeout = TimeSpan.FromSeconds(5);
```

Comment:
Interesting.. I could have sworn this was being used in the earlier versions of the SDK at least.